detox/test/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-all.zip
```

Comment:
The project does not work locally on my machine unless I bump the Gradle version. 🤷 

Otherwise, I just keep seeing this:
https://stackoverflow.com/questions/61289461/java-lang-noclassdeffounderror-could-not-initialize-class-org-codehaus-groovy-v

Reply:
You can upgrade to gradle 6.5+ but need to upgrade the android gradle plugin to 4.1.0+
https://developer.android.com/studio/releases/gradle-plugin